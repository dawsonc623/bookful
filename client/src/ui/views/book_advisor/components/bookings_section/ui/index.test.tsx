import React from "react";

import {
  render
} from "@testing-library/react";

import "@testing-library/jest-dom/extend-expect";

import BookingsSection from ".";

import advisorBookingCollectionFactory from "../../../../../../app/advisor_booking_collection/factory";

describe("BookingsSection", () => {
  test("Renders without error", () => {
    const advisorBookingCollection = advisorBookingCollectionFactory.construct([]);

    const {
      asFragment
    } = render(
      <BookingsSection
        advisorBookings = {advisorBookingCollection}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});