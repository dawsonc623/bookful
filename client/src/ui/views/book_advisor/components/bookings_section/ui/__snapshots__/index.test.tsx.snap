// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookingsSection Renders without error 1`] = `
<DocumentFragment>
  <div
    class="bookingsSection"
  >
    <div
      class="title"
    >
      <span
        class="mdc-typography--headline5"
      >
        Booked Times
      </span>
    </div>
    <div>
      <hr />
    </div>
    <div
      class="bookingsTable mdc-data-table rmwc-data-table--sticky-rows rmwc-data-table--sticky-rows-1"
    >
      <table
        class="mdc-data-table__table"
      >
        <thead
          class="rmwc-data-table__head"
        >
          <tr
            class="rmwc-data-table__row mdc-data-table__header-row"
          >
            <th
              class="tableColumnConstrained rmwc-data-table__cell mdc-data-table__header-cell"
              role="columnheader"
              scope="col"
            >
              Advisor ID
            </th>
            <th
              class="tableColumnConstrained rmwc-data-table__cell mdc-data-table__header-cell"
              role="columnheader"
              scope="col"
            >
              Student Name
            </th>
            <th
              class="rmwc-data-table__cell mdc-data-table__header-cell"
              role="columnheader"
              scope="col"
            >
              Date/Time
            </th>
          </tr>
        </thead>
        <tbody
          class="mdc-data-table__content"
        >
          <tr
            class="rmwc-data-table__row mdc-data-table__row"
          >
            <td
              class="mdc-data-table__cell rmwc-data-table__cell"
            />
            <td
              class="mdc-data-table__cell rmwc-data-table__cell"
            >
              No Bookings
            </td>
            <td
              class="mdc-data-table__cell rmwc-data-table__cell"
            />
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</DocumentFragment>
`;
